sunpy/io/ana.py

```diff
@@ -21,26 +21,25 @@
 except ImportError:
     _pyana = None
 
+ANA_DEPRECATION_MESSAGE = (
+    "The ANA reader may be removed in a future version of sunpy, "
+    "please comment here if you are using this code: "
+    "https://community.openastronomy.org/t/possible-deprecation-of-ana-file-readers-and-writers-in-sunpy"
+)
+
+__all__ = ['read', 'get_header', 'write']
+
 
 def check_ana_installed(func):
     ana_not_installed = (
         "C extension for ANA is missing. For more details see: "
         "https://docs.sunpy.org/en/stable/installation.html#installing-without-conda"
     )
-    if _pyana is None:
+    if _pyana is None and os.environ.get("SUNPY_NO_BUILD_ANA_EXTENSION") is None:
```

Comment:
How would we ever make it to the second clause here? Surely if `NO_BUILD` is true then we wont have built it?

Reply:
It hasn't compiled the extension when we run the sdist tests?!

Reply:
No, why would it? We disable it in the workflow.